lib/vmdb/config/activator.rb

```diff
@@ -22,6 +22,10 @@ def log(data)
         Vmdb::Loggers.apply_config(data)
       end
 
+      def ntp(_data)
+        MiqServer.my_server.ntp_reload_queue unless MiqServer.my_server.nil? rescue nil
```

Comment:
I don't know why the rescue nil is there in the first place.

I am planning on :scissors: activator and validator anyway with config revamp, which is why I'm not interested in researching the why.